main.py

```diff
@@ -2,6 +2,41 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor
+from db.models import Genre
```

Comment:
You can make it in one import separated by coma

Reply:
agree, fix this